genbadge/utils_flake8.py

```diff
@@ -115,7 +117,14 @@ def parse_flake8_stats(stats_txt  # type: str
     for line in stats_txt.splitlines():
         match = RE_TO_MATCH.match(line)
         if not match:
```

Comment:
Now that there are several cases, it would probably be preferable to first handle the "match" (reverting the if). That would put the nominal case upfront.

Reply:
Pardon me for the delay, I looked into it a little, and just had a chance to go back.  I was concerned about a variety of things, including exactly how find_severity worked.  I added my flake8stats.txt and modified the tests to make sure they could pass with it.